rcgen/src/key_pair.rs

```diff
@@ -388,15 +388,6 @@ impl KeyPair {
 		}
 	}
 
-	/// Get the raw public key of this key pair
-	///
-	/// The key is in raw format, as how [`ring::signature::KeyPair::public_key`]
-	/// would output, and how [`ring::signature::UnparsedPublicKey::verify`]
-	/// would accept.
-	pub fn public_key_raw(&self) -> &[u8] {
```

Comment:
this function is useful, please don't remove it.

Reply:
It's still there as `<KeyPair as PublicKeyData>::public_key_der()`.

Reply:
I ended up dropping this commit for now.